packages/models/poll_model/lib/src/poll.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:meta/meta.dart';
+
+part 'poll.g.dart';
+
+enum PollModelType {
```

Comment:
Naming: its not a poll itself actually, its a question from a poll. Worth renaming?

Reply:
Actually, it is a single poll which has type and united to one big user poll. Both poll and poll_question are solid. But first is shorter.